admin-dev/themes/default/js/bundle/product/product-combinations.js

```diff
@@ -137,12 +139,41 @@ var combinations = (function() {
           }
         }, 'input[id^="combination"][id$="_attribute_price"]')
 
+        .on({
+          // when typing a new ecotax on the form, update it on the final price data attribute
+          'keyup': function () {
+            var attributeId = $(this).closest('.combination-form').attr('data');
+            var finalPriceLabel = getCombinationRow(attributeId).find('.attribute-finalprice span');
+            var combinationEcotax = priceCalculation.getCombinationEcotaxTaxExcluded($(this));
+            if (combinationEcotax <= 0) {
+              combinationEcotax = priceCalculation.getProductEcotaxTaxExcluded();
+            }
+
+            finalPriceLabel.data('productEcotax', combinationEcotax);
```

Comment:
Could you factorize this code ?

Reply:
It was removed mostly after a bug fix ^^ better get the data from the input JIT rather than maintaining data attributes on the whole page